Bizyhood/Seo.php

```diff
@@ -0,0 +1,276 @@
+<?php
+
+// Make sure we don't expose any info if called directly
+if ( !function_exists( 'add_action' ) ) {
+	echo 'Hi there!  I\'m just a plugin, not much I can do when called directly.';
+	exit;
+}
+
+/**
+ * This class contains all Bizyhood Shortcodes.
+ */
+ 
+class Bizyhood_Seo
```

Comment:
This should be changed to Sitemap. I don't think the class needs the Bizyhood_ prefix. Seo is a bit too broad a description for what we're doing here.